cocotb/_build_libs.py

```diff
@@ -239,12 +263,13 @@ def _get_common_lib_ext(include_dir, share_lib_dir, sim_define):
     #  simulator
     #
     libsim = Extension(
-        os.path.join("cocotb", "libs", sim_define.lower(), "simulator"),
+        os.path.join("cocotb", "simulator"),
         include_dirs=[include_dir],
         libraries=[_get_python_lib_link(), "cocotbutils", "gpilog", "gpi"],
         library_dirs=python_lib_dirs,
         sources=[os.path.join(share_lib_dir, "simulator", "simulatormodule.c")],
         extra_compile_args=_extra_cc_compile_args,
+        extra_link_args=["-Wl,-rpath,$ORIGIN/libs"],
```

Comment:
Possibly:

```suggestion
        runtime_library_dirs=["$ORIGIN/libs"],
```

Reply:
It should work but it does not work (fails to do what support to do on Linux and crashes on Windows).

Reply:
https://travis-ci.org/github/themperek/cocotb/builds/665284062

Reply:
Thanks for testing.

Reply:
It would be great if the CI log ran setup.py in verbose mode.

Reply:
need to add to `tox.ini`
```
install_command =
    python -m pip install -v {opts} {packages}
``` 

and call `tox -vv`

Reply:
Mind making a separate PR to do just that?

Reply:
Opened #1530 and #1525